ruling/.eslintignore

```diff
@@ -0,0 +1 @@
+!**/node_modules/**
```

Comment:
Ruling used to analyse files in `**/node_modules/**`, but [ESLint 7.x now ignores that pattern by default](https://eslint.org/docs/user-guide/migrating-to-7.0.0#default-ignore-patterns-have-changed). We thus explicitly tell ESLint not to ignore that pattern with a `.eslintignore` file.